#
# K2HDKC DBaaS based on Trove
#
# Copyright 2020 Yahoo Japan Corporation
#
# K2HDKC DBaaS is a Database as a Service compatible with Trove which
# is DBaaS for OpenStack.
# Using K2HR3 as backend and incorporating it into Trove to provide
# DBaaS functionality. K2HDKC, K2HR3, CHMPX and K2HASH are components
# provided as AntPickax.
# 
# For the full copyright and license information, please view
# the license file that was distributed with this source code.
#
# AUTHOR:
# CREATE:
# REVISION:
#

diff --git a/trove/common/template.py b/trove/common/template.py
index 034e3c1a..5589e16a 100644
--- a/trove/common/template.py
+++ b/trove/common/template.py
@@ -38,6 +38,7 @@ SERVICE_PARSERS = {
     'redis': configurations.RedisConfParser,
     'vertica': configurations.VerticaConfParser,
     'db2': configurations.DB2ConfParser,
+    'k2hdkc': configurations.K2hdkcConfParser,
 }
 
 

#
# Local variables:
# tab-width: 4
# c-basic-offset: 4
# End:
# vim600: noexpandtab sw=4 ts=4 fdm=marker
# vim<600: noexpandtab sw=4 ts=4
#
